scripts/version.sh

```diff
@@ -12,7 +12,4 @@ set -euo pipefail
 
 VERSION="${VERSION:-"$(sed -n 's/^version\s=\s"\(.*\)"/\1/p' Cargo.toml)"}"
 CHANNEL="${CHANNEL:-"$(scripts/release-channel.sh)"}"
-if [ "$CHANNEL" == "nightly" ]; then
-  VERSION="$VERSION-nightly"
-fi
```

Comment:
Is this intentional? If so, we should remove the `CHANNEL` var as well.

Reply:
So ... I wish life was so simple. We use the `CHANNEL` environment variable in several places to direct packages to the night place.